satpy/readers/seviri_l1b_native.py

```diff
@@ -307,11 +346,47 @@ def get_area_extent(self, dsid):
         # When dealing with HRV channel and full disk, area extent is
         # in two pieces
         if (dsid.name == 'HRV') and self.mda['is_full_disk']:
-            # NOTE: Implement HRV full disk area_extent
-            # NotImplementedError does not catch this, it must
-            # be used elsewhere already
-            msg = 'HRV full disk area extent not implemented.'
-            raise RuntimeError(msg)
+
+            center_point = HRV_NUM_COLUMNS / 2
```

Comment:
please move `center_point` calculation under the `if`-statement starting on line 332. `column_step`, and `line_step` could be calculated right after the `if`-statement using the `reference_grid` variable. `column_step` starting on line becomes redundant then.